### PR TITLE
update portfolio desktop version.

### PR DESCRIPTION
**Pull request by sameh080081**
this pull request is to update the portfolio with the desktop version.

### CODE REVIEW

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @sameh080081, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request



You have submitted a project, that:
- [ ] Commit messages should be meaningful: it is not something that you can change now, but we need to commit after every feature implemented on our code with meaningful commits written in imperative tense with a capital letter;
- [ ] Pull Request should have a short summary describing introduced changes in more detail.
- [ ] `node_modules` folder should be in the `.gitignore`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by sameh080081** [comment]
Hi
Thank you, could please elaborate I still do not get your point.
I don't know exactly what is wrong.

**Comment by nicupop729** [comment]
Hi @sameh080081,
Thanks for reaching out!

As explained above, this pull request holds a very poor commit history, badly named (commits without a capital letter, with lack of meaning, and very few commits). It is not something you can do about now, rather is just a heads up for you to pay more attention to this in the future.
As well, the pull request had a missing description. In the description, we need to explain more in detail the changes we bring to the project with the pull request.
`node_modules` folder was missing from `.gitingnore` when it's mandatory to have it always there.

If still needed, please don't hesitate to reach back.

**Comment by sameh080081** [comment]
Thank you for the clarification.
Still a little bit confused about node modules I think I can search for it.

File: style.css [hunk 1]
```diff
@@ -664,3 +684,189 @@ h3 {
   background-color: #28352f;
   margin-top: 40px;
 }
+
+@media screen and (min-width: 650px) {
+  .page1 {
```
Comment (AmaduKamara) [inline style.css]: - [x] Kindly set your media query to have a min-width of range 768px - 1440px. Any value within that range is ok. I will recommend you use 768px that ranging from tablet sizes to big screens. You are using a smaller number here and for this project, it's required you use a range from 768px or higher.
- [x] Kindly check the requirement and see the range you were asked to use.
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ 

Hi @sameh080081,

SoCloseKeeganMichaelKeyGIF

Good job so far, your project is looking nice!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ No linter error
✅ Project is working fine

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @sameh080081 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**